awscli/examples/mediaconvert/get-job.rst

```diff
@@ -0,0 +1,38 @@
+**To get details for a particular job**
+
+The following example requests the information for the job with ID ``1234567890987-1ab2c3``, which in this example ended in an error. ::
+
+    aws mediaconvert get-job \
+        --endpoint-url=https://abcd1234.mediaconvert.region-name-1.amazonaws.com \
+        --region=region-name-1 \
+        --id=1234567890987-1ab2c3
+
+To get your account-specific endpoint, use ``describe-endpoints``, or send the command without the endpoint. The service returns an error and your endpoint.
+
+Output::
```

Comment:
The double `::` on Output means it's expecting the next line to be an indented literal block, so this is causing rendering issues.

You can either move the paragraph on line 14 above the `Output::` section or just remove the `::` from `Output::`.